### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by yetieaterxb1**
- setup API routes
-seeding script complete (still optimizing)

### CODE REVIEW

File: seedAlsoViewed.js [hunk 1]
```diff
@@ -73,39 +48,49 @@ const getShippingCost = (freeShipping) => {
...
-  if (err) {
-    throw err;
+const seeding = async () => {
+  // ViewedItem.sync({ force: true });
+  console.time('SeedingTime');
+  let seedAlsoViewedItems = [];
+  const startId = 101;
+  const endId = 500000;
+
+  for (let i = startId; i <= endId; i += 1) {
```
Comment (willresen) [inline seedAlsoViewed.js:63]: This may be a great place for a helper function.

File: seedAlsoViewed.js [hunk 2]
```diff
@@ -60,7 +60,7 @@ const seeding = async () => {
   const endId = Number(process.env.END_ITEM_ID) || 10000000;
 
   for (let i = startId; i <= endId; i += 1) {
-    console.log(seedAlsoViewedItems.length)
+    console.log(seedAlsoViewedItems.length);
```
Comment (mchung1128) [inline seedAlsoViewed.js]: Is this console log necessary?

File: server/db/index.js [hunk 3]
```diff
@@ -1,20 +1,24 @@
...
+  oldPrice: Sequelize.DECIMAL,
+  currentPrice: Sequelize.DECIMAL,
+  freeSheeping: Sequelize.BOOLEAN,
+  shippingCost: Sequelize.DECIMAL,
+  categoryId: Sequelize.INTEGER,
+}, { sequelize, modelName: 'vieweditems' });
 
-module.exports = db;
+module.exports = ViewedItem;
+exports.sequelize = sequelize;
```
Comment (willresen) [inline server/db/index.js]: You should add `sequelize` directly to the `module.exports` object.

**Review by willresen — APPROVED** [review]
Looks good!

File: seedAlsoViewed.js (hunk 2 above)
Comment (mchung1128) [reply seedAlsoViewed.js]: Never mind, looks like this was an old commit.

File: server/db/index.js (hunk 3 above)
Comment (yetieaterxb1) [reply server/db/index.js]: Good suggestion

**Review by willresen — APPROVED** [review]

**Pull request merged**